sample/src/main/res/layout/activity_main.xml

```diff
@@ -77,4 +77,11 @@
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:text="The last button"/>
+
+  <ImageView
+      android:id="@+id/image_view"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:src="@mipmap/ic_barista"
```

Comment:
I'm not sure if it's OK to assign a @mipmap image to a ImageView... I think that the mipmap drawables are used by third party apps, like the launchers, to get images in higher density that the expected one. I mean, i think that we need to use a common @drawable resource here...